test/lcov_test.exs

```diff
@@ -53,18 +53,18 @@ defmodule ExCoveralls.LcovTest do
       Lcov.execute(@source_info)
     end) =~ @stats_result
 
-    assert(File.read!(report) =~ ~s(TN:\nSF:test/fixtures/test.ex\nDA:1,0\nDA:2,1\nLF:2\nLH:1\nend_of_record\n))
-    %{size: size} = File.stat! report
-    assert(size == @file_size)
```

Comment:
I had to remove the file size assertion because with the absolute path being used, it would cause the file size to be different.

We already test the file size via the `File.read!` and matching it against the string.